.gitignore

```diff
@@ -1,3 +1,5 @@
 __pycache__
 .pytest_cache
-test
\ No newline at end of file
+test
```

Comment:
I'm assuming this is where cocotb-test puts the verilated files?
Is there a way to set this to some other directory (e.g. `build`?)
Right now this is a bit confusing in my opinion, since there already is a tests folder in the repository that should not be ignored.

Reply:
Yup, I can fix this.